src/function/probability/lgamma.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-loss-of-precision */
+
+import { lgammaNumber, lnSqrt2PI } from '../../plain/number/index.js'
+import { deepMap } from '../../utils/collection.js'
+import { factory } from '../../utils/factory.js'
+
+const name = 'lgamma'
+const dependencies = ['Complex', 'typed']
+
+export const createLgamma = /* #__PURE__ */ factory(name, dependencies, ({ Complex, typed }) => {
+  const TWOPI = 6.2831853071795864769252842 // 2*pi
+  const LOGPI = 1.1447298858494001741434262 // log(pi)
+  const SMALLX = 7
+  const SMALLY = 7
+
+  const REFLECTION = 0.1
+
+  const coeffs = [
+    -2.955065359477124183e-2, 6.4102564102564102564e-3, -1.9175269175269175269e-3, 8.4175084175084175084e-4,
+    -5.952380952380952381e-4, 7.9365079365079365079e-4, -2.7777777777777777778e-3, 8.3333333333333333333e-2
+  ]
+
```

Comment:
Constants SMALLX, SMALLY, REFLECTION, and coeffs should either be commented more fully with their meanings, or defined closer to their point of use, or both.

Reply:
Possibly a more specific name than "coeffs" would be helpful as well.

Reply:
Renamed `SMALLX` (to `SMALL_RE`) and `SMALLY` (to `SMALL_IM`) and provided more info for them and `coeffs`. `REFLECTION` was moved closer to where it's used.

Reply:
Looks fine now.